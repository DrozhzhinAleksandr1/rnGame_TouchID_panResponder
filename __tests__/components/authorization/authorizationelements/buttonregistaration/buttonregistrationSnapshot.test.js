import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import {ButtonRegistration}  from '../../../../../app/components/authorization/authoriztionelements/buttonregistration';

it('create snapshot Main showMenu={true}', () => {
    const renderer = new ShallowRenderer()
    let buttonRegistrationSnapshot = renderer.render(<ButtonRegistration />);
    expect(buttonRegistrationSnapshot).toMatchSnapshot();
});