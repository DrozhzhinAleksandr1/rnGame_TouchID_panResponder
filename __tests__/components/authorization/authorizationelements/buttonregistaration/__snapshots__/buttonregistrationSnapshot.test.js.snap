// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot Main showMenu={true} 1`] = `
<TouchableWithoutFeedback
  accessibilityLabel="BtnRegistrationAuthorizationID"
  onPress={[Function]}
  testID="BtnRegistrationAuthorizationID"
>
  <Component>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#4BCEA9",
          "fontFamily": "Roboto-Regular",
          "fontSize": 18,
        }
      }
    />
  </Component>
</TouchableWithoutFeedback>
`;
